edge-hub/test/Microsoft.Azure.Devices.Routing.Core.Test/DispatcherTest.cs

```diff
@@ -20,9 +20,9 @@ namespace Microsoft.Azure.Devices.Routing.Core.Test
     [ExcludeFromCodeCoverage]
     public class DispatcherTest : RoutingUnitTestBase
```

Comment:
Are these test changes just updates to get things to compile with priorities? Or are the test cases being changed?

Reply:
it's to get them to run against the new change.  Previously the test took a shortcut with message offsets, and relied on the fact that the NullCheckpointer implementation ignored the offset values.

Now that there's no option to pass in NullCheckpointer anymore, the message offset is actually being calculated and checked, so the test messages we pass in need to have ascending offsets like real messages.